cmd/dmsg-discovery/internal/api/api.go

```diff
@@ -229,3 +247,13 @@ func (a *API) writeJSON(w http.ResponseWriter, r *http.Request, code int, object
 		a.log(r).Warnf("Failed to write response: %s", err)
 	}
 }
+
+// UpdateInteralState is background function which updates numbers of clients and servers.
+func (a *API) UpdateInteralState(ctx context.Context, logger logrus.FieldLogger) {
+	numberOfServers, numberOfClients, err := a.db.CountEntries(ctx)
+	if err != nil {
+		logger.WithError(err).Errorf("failed to get number of clients and servers")
```

Comment:
In case of error, it wouldn't be seen in metrics returned by the endpoint. What do you think about adding a field that indicates an error?

Reply:
As you added an `error` field, the error here could be written to `a.error`.

Reply:
Thank you.